modules/ocf/manifests/firewall/post.pp

```diff
@@ -20,7 +22,7 @@
       opts => {
```

Comment:
@jvperrin @abizer @chriskuehl @matthew-mcallister I've been wondering if it's a better idea to simply drop packets from DMZed servers (i.e. tsunami, werewolves, corruption) instead. It would make the mesos rules much simpler and other rules unnecessary.